fs/sync.go

```diff
@@ -40,8 +40,14 @@ type syncCopyMove struct {
 
 func newSyncCopyMove(fdst, fsrc Fs, Delete bool, DoMove bool) *syncCopyMove {
 
-	// Don't track renames for remotes without rename support.
-	_, canMove := fdst.(Mover)
+	// Don't track renames for remotes without server-side rename support.
+	// Some remotes simulate rename by server-side copy and delete, so include
+	// remotes that implements either Mover and Copier.
+	var canMove bool
+	switch fdst.(type) {
+	case Mover, Copier:
+		canMove = true
+	}
 
```

Comment:
Tracking renames relies on both the source and dest having a common hash.  You can test for this with

```go
	common := fsrc.Fs().Hashes().Overlap(fdst.Fs().Hashes())
	if common.Count() == 0 {
		// no hashes in common so cantMove
                canMove = false
                // this should probably have a log too
	}
```